ecommerce/coupon_discounts/lib/coupon_discounts/coupon.rb

```diff
@@ -0,0 +1,30 @@
+
+module CouponDiscounts
+  class Coupon
+    include AggregateRoot
+
+    AlreadyRegistered = Class.new(StandardError)
+
+    def initialize(id)
+      @id = id
+    end
+
+    # TODO: do not allow for duplicates name and value coupons
```

Comment:
let's express TODOs as new GH tickets and not use code comments for that

Reply:
Added an issue